tests/tasks/test/integration/prometheus.sh

```diff
@@ -33,6 +33,15 @@ if [[ $response != *"cipherstash_proxy_rows_total 1"* ]]; then
     exit 1
 fi
 
+if [[ $response != *"cipherstash_proxy_statements_execution_duration_seconds{quantile=\"1\"} 0."* ]]; then
```

Comment:
I wish I could think of a more sensible test than this, but after:
1. trying and failing to move this test into rust, and
2. trying and failing to think of a way of instrument everything enough to check this value somehow,

… I've reverted to this is as a good-enough-for-now go at it.